### ISSUE TITLE
Problems to show in front end.

### ISSUE THREAD

**Issue by just-adriandev** [ueberdosis/tiptap-docs#37]
I have a site in nextjs, it's to write some articles in the form of posts. When I'm writing, everything looks very good and well formatted on the page and creation, but when I create the post and see the result, my frontend receives everything incorrectly, the lists in general don't even appear, they are as if they were null, and when I apply bold, strike and other markings, the text goes without any formatting, just the plain text, without being in bold, without being in h1 and so on.

**Comment by EdgarsJoja** [comment]
Just a wild guess - are you using TailwindCSS or similar on FE? Tailwind comes with preflight (CSS reset), so it removes default styles - lists, bold, borders, spacing, etc. It's something you have to add back yourself (or remove preflight: https://tailwindcss.com/docs/preflight)

**Comment by just-adriandev** [comment]
yes, I'm using talwind to style, I'm going to try and come back here to expose the result.
